packages/node_modules/@ciscospark/widget-recents/src/components/PopoverContent.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+import {Avatar} from '@collab-ui/react';
+
+import styles from './PopoverContent.css';
+
+
+const propTypes = {
+  currentUser: PropTypes.shape({
+    id: PropTypes.string,
+    displayName: PropTypes.string,
+    email: PropTypes.string,
+    img: PropTypes.string
+  }).isRequired,
+  onSignOutClick: PropTypes.func
+};
+
+const defaultProps = {
+  onSignOutClick: () => {}
+};
+
+
+function PopoverContent(props) {
+  const {
+    currentUser,
+    onSignOutClick
+  } = props;
+
+  /* eslint-disable jsx-a11y/click-events-have-key-events */
```

Comment:
Why are we disabling accessibility features?

Reply:
Signout div still needs keyboard handling

Reply:
we didn't talk about adding a keyboard handling to signout. and Other clients don't even support that. I don't see a point adding a keyboard handling to signout. Unless its just a an empty function.